tests/nvmeof-mp/004

```diff
@@ -7,6 +7,10 @@
 DESCRIPTION="File I/O on top of multipath concurrently with logout and login (sq-on-mq)"
 TIMED=1
 
+requires() {
+	have_legacy_dm || return $?
```

Comment:
The `|| return $?` isn't necessary here and everywhere else that it's the last command in a function.